src/common/ErrorPage/index.js

```diff
@@ -1,6 +1,6 @@
 import { Link } from "react-router-dom";
 import { toMovies } from "../../core/App/routes";
-import { ErrorWrapper, ErrorMessage, ErrorInfo, Button, ErrorIcon } from "./styled";
+import { ErrorWrapper, ErrorMessage, ErrorInfo, Button, ErrorIcon,} from "./styled";
```

Comment:
hey you know what I think the comma at the end is unnecessary :D But overall it's good :D